src/main/java/org/team1540/robot2025/subsystems/intake/IntakeConstants.java

```diff
@@ -0,0 +1,23 @@
+package org.team1540.robot2025.subsystems.intake;
+
+public class IntakeConstants {
+
+    // TODO: tune
+
+    public static final int SPIN_FALCON_ID = 0;
+    public static final int PIVOT_FALCON_ID = 1;
+    public static final int NEO_ID = 2;
+
+    public static final double SPIN_FALCON_CURRENT_LIMIT = 10;
+    public static final double PIVOT_FALCON_CURRENT_LIMIT = 10;
+
+    public static final double INTAKE_KS = 0.25;
+    public static final double INTAKE_KV = 0.12;
+    public static final double INTAKE_KA = 0.01;
+    public static final double INTAKE_KP = 2.4;
+    public static final double INTAKE_KI = 0;
+    public static final double INTAKE_KD = 0.1;
+
+    public static final int INTAKE_CRUISE_VELOCITY = 80;
+    public static final int INTAKE_ACCELERATION = 160;
```

Comment:
It's unclear what part of the intake these constants refer to. Should probably be renamed PIVOT_... instead.